### PR TITLE
Alex bump dependencies

### PR DESCRIPTION
**Pull request by bugsalexander**
Bump all dependencies to their latest versions.

Had some minor compatibility versions, the most being related to Gatsby deprecating their old image plugin `gatsby-image`, and replacing it with the new `gatsby-plugin-image`. 

The changes can be found [here](https://www.gatsbyjs.com/docs/reference/release-notes/image-migration-guide), but I will summarize the changes I made in response to the migration:

# Gatsby Image Plugin Migration Changes

### Import Syntax
```javascript
// import Img from 'gatsby-image';
import { GatsbyImage } from 'gatsby-plugin-image';
```

### Component Property Naming
```jsx
// <Img fixed={data.file.childImageSharp.fixed} />
<GatsbyImage image={data.file.childImageSharp.gatsbyImageData} />
```

### GraphQL Changes
Some GraphQL properties became parameters. Idk if this is the right verbiage, but something like this:
```graphql
childImageSharp {
  fixed {
    ...GatsbyImageSharpFixed
  }
}
```
turned into
```graphql
childImageSharp {
  gatsbyImageData(layout: FIXED)
}
```

## Compatibility with gatsby-background-image
the plugin `gatsby-background-image` still uses the old image format (not the above) and so we need to make sure we're compatible with it. this information can be found here: https://www.gatsbyjs.com/plugins/gatsby-background-image/#gatsby-3--gatsby-plugin-image

Basically, the fix is you add a new package that adds compatibility.

# Amplitude Fix

for some reason, the import path for amplitude changed from `amplitude-js/amplitude` to just `amplitude`. Not really sure why, but if you try it with the `/amplitude` it should crash, and it doesn't without it.

### CODE REVIEW

File: src/components/TeamPage/teamPhotos.js [hunk 1]
```diff
@@ -60,14 +60,17 @@ const ProfileRole = styled.span`
...
   picture img {
     transition: filter ease-in 200ms !important;
   }
 `
 
 const ProfileImgWrapper = styled.div`
   position: relative;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 4.2px;
```
Comment (epillay) [inline src/components/TeamPage/teamPhotos.js:73]: I think this is where the random 4.2 margin is coming from 🔍

Comment (bugsalexander) [reply src/components/TeamPage/teamPhotos.js:73]: oh yeah haha i added this in because it was not centered and was missing 4.2px before

**Review by epillay — APPROVED** [review]
